janus_server/src/client.rs

```diff
@@ -29,63 +37,215 @@ static CLIENT_USER_AGENT: &str = concat!(
     "client"
 );
 
+pub async fn aggregator_hpke_sender(
```

Comment:
This function and `default_http_client` used to be associated functions on `struct Client`. I moved them up to the module so that they could be used without worrying about the generic type parameters on `Client`, since none of those are relevant to these functions.